tests/phpunit/tests/mail/test-phpmailer-translations.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+class Test_PHPMailer_Translations extends WP_UnitTestCase {
+	/**
+	 * @ticket 23311
+	 */
+	public function test_wp_phpmailer_error_message_keys_match() {
+		require_once ABSPATH . WPINC . '/PHPMailer/PHPMailer.php';
+		require_once ABSPATH . WPINC . '/class-wp-phpmailer.php';
```

Comment:
We should remove these lines and instead update `tests/phpunit/includes/mock-mailer.php` to instantiate `WP_PHPMailer`